pkg/container/types/tuple.go

```diff
@@ -747,6 +747,10 @@ func decodeTuple(b []byte) (Tuple, int, error) {
 		case b[i] == stringTypeCode:
 			el, off = decodeBytes(b[i+1:])
 			off += 1
+		case b[i] == enumCode:
+			//TODO: need to verify @YANGGMM
+			el, off = decodeUint(uint16Code, b[i+1:])
```

Comment:
@YANGGMM is this the correct decode impl for Enum?

Reply:
It should be fine.